Dockerfile

```diff
@@ -3,3 +3,8 @@ FROM php:7.4-fpm-alpine
 WORKDIR /var/www/html
 
 RUN docker-php-ext-install pdo pdo_mysql
+
+# install and enable xdebug
+RUN apk add --no-cache $PHPIZE_DEPS \
+	&& pecl install xdebug-2.7.0 \
```

Comment:
for PHP 7.4 you need 2.9.7

Reply:
Updated to 2.9.8 which is the most recent version thanks! :+1: